### ISSUE TITLE
@example not handled properly in case of indentation

### ISSUE THREAD

**Issue by sagiegurari** [jsdoc2md/jsdoc-to-markdown#35]
although i'm using the grunt plugin to run this, i'm pretty sure the issue is in this library.
Take a look at the file: https://github.com/sagiegurari/angular-web-notification/blob/master/angular-web-notification.js
at the end there is a example tag and due to the fact that the whole js code is indented properly, the doc is also indented and the entire content of the example in the md gets indented too.
that causes the example generated part in the md not to exit from example mode since the ````` tags are indented.

In other words instead of 
```js
something here....
 ```

It generates to:
```js
             something here....
              ```

**Comment by 75lb** [comment]
you need the `*` at the beginning of each line, including throughout the @example code.. it should look like this
```
/**
 * Shows the notification based on the provided input.<br>
 * The callback invoked will get an error object (in case of an error, null in
 * case of no errors) and a 'hide' function which can be used to hide the notification.
 *
 * @function
 * @memberof! webNotification
 * @alias webNotification.showNotification
 * @public
 * @param {string} [title] - The notification title text (defaulted to empty string if null is provided)
 * @param {object} [options] - Holds the notification data (web notification API spec for more info)
 * @param {function} callback - Called after the show is handled.
 * @example
 * webNotification.showNotification('Example Notification', {
 *    body: 'Notification Text...',
 *    icon: 'my-icon.ico'
 * }, function onShow(error, hide) {
 *    if (error) {
 *        window.alert('Unable to show notification: ' + error.message);
 *    } else {
 *        setTimeout(function hideNotification() {
 *            hide();
 *        }, 5000);
 *    }
 * });
 */
```

**Comment by sagiegurari** [comment]
that actually used to create other problems i remember.
guess things change.
I'll try it out.
